README.md

```diff
@@ -199,6 +199,32 @@ aa('getUserToken', null, (err, userToken) => {
 });
 ```
 
+#### Listen to `userToken` change
+
+If you want to attach a listener for `userToken` change, you can call `onUserTokenChange`.
+
+```js
+aa('onUserTokenChange', (userToken) => {
+  console.log("userToken has changed: ", userToken);
+});
+```
+
+`onUserTokenChange` accepts `callback`(required) and `options`(optional).
+
+```js
+aa('onUserTokenChange', callback, options);
+```
+
+| Option      | Type       | Description                                    |
+| ----------- | ---------- | ---------------------------------------------- |
+| `immediate` | `boolean`  | Fire the callback as soon as it's attached     |
+
+If there is a chance that userToken is already set before `onUserTokenChange`, then you can provide `{ immediate: true }` option in order to get the token as soon as you attach `onUserTokenChange`.
```

Comment:
Instead of "If there is a chance that [...]", can we give an example when this can happen? It might get even more confusing for users otherwise.

Reply:
thanks. wdyt? https://github.com/algolia/search-insights.js/pull/209/commits/76304708bf4a800fdef4e34327b25b5eeae39f37